packages/forma-36-react-components/src/components/Dropdown/DropdownContainer/DropdownContainer.css

```diff
@@ -22,6 +22,11 @@
   margin: 0;
 
   z-index: var(--z-index-dropdown);
+
+  &:focus {
+    outline: none;
+    box-shadow: var(--glow-muted), var(--box-shadow-default);
```

Comment:
@mshaaban0 let's use `glow-primary` here. We will change the Secondary and Transparent buttons' focus glow to use `glow-primary`, too.

Reply:
Thanks for the fix!

Reply:
@domarku Updated